script/Ciel/Day 9/09_01_CIEL09_2C.txt

```diff
@@ -6,67 +6,71 @@ Classes are over for the day, but I'm still in a bit of a somber mood.
 }
 [sha:166be0a9b15558ea662b31d127e2760313eb51f8]{
 -- Page 10007, Offset 22836.
 -- 通り魔事件の影響で放課後の部活動も自粛中だ。大部分の生徒は帰宅し、校舎は静まりかえっている。
-After-school club activities have been suspended because of the serial murders. The vast majority of students have already gone home, leaving the schoolyard completely silent.
+After-school club activities have been suspended because of the serial murders.
+ Almost all of the students have already gone home, leaving the school building as silent as the grave.
 }
 [sha:1c5f9e7a6d8cb87debc5d027cb8a5cfdb532d5f2]{
 -- Page 276, Offset 22837.
 -- ……絶交の言葉通り、先輩がやってくる気配は微塵もない。
-Senpai said she never wanted to see me again. It seems like she meant it, considering there aren't any signs of her coming to pay me a visit.
+It doesn't look like Senpai's coming to pay me a visit...
+ I guess she meant it when she said she never wanted to see me again.
 }
 [sha:84f88cb3cf5f997c203bf8ecc7011121fb022916]{
 -- Page 277, Offset 22838.
 -- 「い、いや、別に気にしてないし。
-"N-No, it's no big deal anyway.
+// Modified a bit since a) this is still plenty tsundere and b) it connects more clearly to the next line
+"I-I mean, it's not like I was waiting for her or anything.
 }
 [sha:c9b935e9c60c215964f5bdde74bbe3396dd210f2]{
 -- Page 277, Offset 22839.
 -- たんに休んでいただけだし、もうさっさと帰るからな」
-I was just taking a quick break, but it's about time to head home."
+I was just taking a quick break before I set off back to the mansion."
 }
 [sha:660aa65eb4d5d6110196a909b2548e4c69dd80eb]{
 -- Page 277, Offset 22840.
 -- 誰に言い聞かせるでもなく呟く。
-I mutter to no one in particular.
+I mutter aloud to the empty classroom.
 }
 [sha:92ee5986ff5e357c7a542dd6f8f67a1a0e2ae49d]{
 -- Page 278, Offset 22841.
 -- 「メール？」
-"A message?"
+"A text?"
 }
 [sha:d5c38589dff84993928206e2193d04a2d7a0e538]{
 -- Page 278, Offset 22842.
 -- 屋敷に帰ろうと教室を出た時、携帯に着信があった。
-As I leave the classroom in order to make my way home, I receive a phone message.
+// Vibrate only man based on sfx
+As I leave the classroom to head home, I hear my phone vibrate.
 }
 [sha:52e2e2bd6c09f19293cfad7c8bdfc12dacb57b00]{
 -- Page 278, Offset 22843.
 -- 鞄から携帯を取り出す。
-I grab my phone from my bag.
+I reach into my bag and pull it out.
 }
 [sha:f804e60f3ff4cc4ef5539a213b8e2ae723bad986]{
 -- Page 278, Offset 22844.
 -- メールの差出人は意外な事にノエル先生だった。
-Unexpectedly, the message is from Noel-sensei.
+Surprisingly, the text seems to be from Noel-sensei.
 }
 [sha:1e438a0845bbd88a91a9c47937457b8bf891eb6c]{
 -- Page 279, Offset 22845.
 -- 内容は簡素なもので、
-Its content is simple:
+Its contents are brief:
 }
 [sha:6c7230efef0da2b25616e024ca6063033925134a]{
 -- Page 279, Offset 22846.
 -- “時間ある？　見せたい物があるから来てほしいの。
-'Are you free? I have something I'd like to show you, so could you drop by?
+'You free? I've got something I want to show you, so can you come over?
 }
 [sha:17451a7c983c4d17dab1a00775f4647f7f688b0b]{
 -- Page 279, Offset 22847.
 -- もちろんシエルさんにはナイショでね♡”
-Of course, keep this a secret from Ciel-san, okay♡'
+Oh and no telling Ciel-san, okay ♡'
```

Comment:
Looks like we've never had a question end with a heart before, so I guess there's no precedent. I think adding a ? is fine here, esp since this is sms-go and having a heart on the end actually makes sense for once